velox/common/memory/tests/AllocationTest.cpp

```diff
@@ -85,25 +85,27 @@ TEST_F(AllocationTest, appendMove) {
   allocation.clear();
 }
 
-TEST_F(AllocationTest, multiplePageRuns) {
```

Comment:
Please support this test.

Reply:
The test doesn't apply anymore as we won't generate a plan with more than max page run limit.